src/OpenLoco/Objects/ObjectManager.cpp

```diff
@@ -1267,8 +1346,9 @@ namespace OpenLoco::ObjectManager
             return false;
         }
 
-        if (_totalNumImages >= 266266)
+        if (_totalNumImages >= Gfx::G1ExpectedCount::kObjects + Gfx::G1ExpectedCount::kDisc)
```

Comment:
I'm not sure how object loading really works, but is there any reason this doesn't take the steam object count into consideration?

Reply:
The steam image count is just something that matters for loading. It's because the steam version uses an out of date revision of locomotion